exopy_hqc_legacy/instruments/drivers/visa/tektro_awg.py

```diff
@@ -774,8 +817,18 @@ def delete_all_waveforms(self):
         """Deletes all user-defined waveforms from the currently loaded setup
 
         """
+        try:
+            # Number of user defined waveforms
+            nb_waveforms = int(self.ask("WLIST:SIZE?")) - 25
```

Comment:
I would like to see some explanation about the magic number (looks like the predefined waveforms).